tests/README.md

```diff
@@ -70,28 +91,35 @@ make tests args="[TESTFILE]::[TESTNAME]"
 where `TESTFILE` is as above, and `TESTNAME` is the name of the test
 within the test file (ex: `test_elf.py::test_virtual_valid`).
 
-See [pytest docs](https://docs.pytest.org/en/7.1.x/how-to/usage.html)
-for more invocation options.
+To run a single case of a single test:
+
+```sh
+# root of the repo
+make tests args="[TESTFILE]::[TESTNAME][test-case]"
+```
+
+ex: `make tests args="test_elf.py::test_virtual_valid[copy_files0]"`
```

Comment:
can we move example into code block above. it will render better vs being inlined